packages/malloy-db-publisher/src/client/api.ts

```diff
@@ -0,0 +1,2342 @@
+     * @memberof BigqueryConnection
+     */
+    'maximumBytesBilled'?: string;
+    /**
+     * 
+     * @type {string}
+     * @memberof BigqueryConnection
+     */
+    'queryTimeoutMilliseconds'?: string;
+}
+/**
+ * Malloy model def and result data.  Malloy model def and result data is Malloy version depdendent.
+ * @export
+ * @interface CompiledModel
+ */
+export interface CompiledModel {
+    /**
+     * Model\'s package Name
+     * @type {string}
+     * @memberof CompiledModel
+     */
+    'packageName'?: string;
+    /**
+     * Model\'s relative path in its package directory.
+     * @type {string}
+     * @memberof CompiledModel
+     */
+    'path'?: string;
+    /**
+     * Type of malloy model file -- source file or notebook file.
+     * @type {string}
+     * @memberof CompiledModel
+     */
+    'type'?: CompiledModelTypeEnum;
+    /**
+     * Version of the Malloy compiler that generated the model def and results fields.
+     * @type {string}
+     * @memberof CompiledModel
+     */
+    'malloyVersion'?: string;
+    /**
+     * Data style for rendering query results.
+     * @type {string}
+     * @memberof CompiledModel
+     */
+    'dataStyles'?: string;
+    /**
+     * Malloy model def.
+     * @type {string}
+     * @memberof CompiledModel
+     */
+    'modelDef'?: string;
+    /**
+     * Array of model sources.
+     * @type {Array<Source>}
+     * @memberof CompiledModel
+     */
+    'sources'?: Array<Source>;
+    /**
+     * 
+     * @type {Array<Query>}
+     * @memberof CompiledModel
+     */
+    'queries'?: Array<Query>;
+    /**
+     * Array of notebook cells.
+     * @type {Array<NotebookCell>}
+     * @memberof CompiledModel
+     */
+    'notebookCells'?: Array<NotebookCell>;
+}
+
+export const CompiledModelTypeEnum = {
+    Source: 'source',
+    Notebook: 'notebook'
+} as const;
+
+export type CompiledModelTypeEnum = typeof CompiledModelTypeEnum[keyof typeof CompiledModelTypeEnum];
+
+/**
+ * 
+ * @export
+ * @interface Connection
+ */
+export interface Connection {
+    /**
+     * 
+     * @type {string}
+     * @memberof Connection
+     */
+    'name'?: string;
+    /**
+     * 
+     * @type {string}
+     * @memberof Connection
+     */
+    'type'?: ConnectionTypeEnum;
+    /**
+     * 
+     * @type {ConnectionAttributes}
+     * @memberof Connection
+     */
+    'attributes'?: ConnectionAttributes;
+    /**
+     * 
+     * @type {PostgresConnection}
+     * @memberof Connection
+     */
+    'postgresConnection'?: PostgresConnection;
+    /**
+     * 
+     * @type {BigqueryConnection}
+     * @memberof Connection
+     */
+    'bigqueryConnection'?: BigqueryConnection;
+    /**
+     * 
+     * @type {SnowflakeConnection}
+     * @memberof Connection
+     */
+    'snowflakeConnection'?: SnowflakeConnection;
+    /**
+     * 
+     * @type {TrinoConnection}
+     * @memberof Connection
+     */
+    'trinoConnection'?: TrinoConnection;
+}
+
+export const ConnectionTypeEnum = {
```

Comment:
super curious why this list is edited, and how someone who added a new dialect would know to get on this list